packages/camera/camera_platform_interface/lib/src/types/video_capture_options.dart

```diff
@@ -43,6 +44,17 @@ class VideoCaptureOptions {
   /// Should only be set if a streamCallback is also present.
   final CameraImageStreamOptions? streamOptions;
 
+  /// **Android only.**
+  ///
+  /// When `true`, configures the recording to be a persistent recording.
+  ///
+  /// A persistent recording will only be stopped by explicitly calling [CameraController.stopVideoRecording] and will ignore events that would normally cause recording to stop, such as
+  /// * lifecycle events
+  /// * calling [CameraController.setDescription] while a recording is in progress
+  ///
+  /// Defaults to `false`.
```

Comment:
I noted this elsewhere but can you clarify in the docs that you need to set this in order to set description while recording on Android?